lib/SIL/SILOwnershipVerifier.cpp

```diff
@@ -121,20 +121,55 @@ static bool isGuaranteedForwardingValueKind(SILNodeKind K) {
   }
 }
 
-static bool isGuaranteedForwardingValue(SILValue V) {
+static bool isGuaranteedForwardingValue(SILValue v) {
   return isGuaranteedForwardingValueKind(
-      V->getKindOfRepresentativeSILNodeInObject());
+      v->getKindOfRepresentativeSILNodeInObject());
 }
 
-static bool isGuaranteedForwardingInst(SILInstruction *I) {
-  return isGuaranteedForwardingValueKind(SILNodeKind(I->getKind()));
+bool swift::isGuaranteedForwardingInst(SILInstruction *i) {
+  return isGuaranteedForwardingValueKind(SILNodeKind(i->getKind()));
 }
 
 LLVM_ATTRIBUTE_UNUSED
 static bool isOwnershipForwardingInst(SILInstruction *I) {
   return isOwnershipForwardingValueKind(SILNodeKind(I->getKind()));
 }
 
+bool swift::getUnderlyingBorrowIntroducers(SILValue inputValue,
+                                           SmallVectorImpl<SILValue> &out) {
+  SmallVector<SILValue, 32> worklist;
+  worklist.emplace_back(inputValue);
```

Comment:
[comment] I think it's fine if you always use emplace_back, but some people might say it's silly because you're just calling the same SILValue copy constructor as push_back.